Makefile

```diff
@@ -48,18 +45,13 @@ $(if $(filter TRUE,$(or $(BUILD_CORE),TRUE)), $(CORE_COMMAND))
 $(if $(filter TRUE,$(or $(BUILD_SUBSTRAIT),TRUE)), $(SUBSTRAIT_COMMAND))
 endef
 
-.PHONY: duckdb
-duckdb:
-	rm -rf duckdb
-	git clone -b $(DUCKDB_BRANCH) --depth 1 $(DUCKDB_REPO)
-
 .PHONY: substrait
 substrait:
 	rm -rf substrait
 	git clone -b $(SUBSTRAIT_BRANCH) --depth 1 $(SUBSTRAIT_REPO) --recurse-submodules substrait
 
 .PHONY: deps.header
-deps.header: duckdb substrait
```

Comment:
In order to use duckdb at a specific hash, I had to add it as a submodule . Removed the duckdb git clone in Makefile.